### ISSUE TITLE
Examples of rendering custom components using Zod

### ISSUE THREAD

**Issue by j1mie** [m6io/react-formgen#3]
I'm looking for some examples of how `@react-formgen/zod` can be used to map different Zod types to render custom components — do you have any examples of this? All examples seem to be based on the JSON schema.

For additional context I am using this in a [react-native-web](https://necolas.github.io/react-native-web/) project where I'll be rendering custom form components.

Thanks!

**Comment by m6io** [comment]
Yeah I primarily use the JSON Schema implementation myself due to it's versatility and metadata, but I'll try and add some more Zod ones. Zod is a bit tricky to work with in this context because it doesn't really have a lot of standard built-in metadata, but I'll try and come up with something!

If you have some samples of the zod schemas you're looking to use and the use case you're thinking of, feel free to drop them here.

**Comment by NotFiliberto** [comment]
any news about zod implementation?

**Comment by tobimori** [comment]
Might be interesting: https://v4.zod.dev/metadata

**Comment by m6io** [comment]
Oh, and please note: I haven't updated the doc yet so I'm going to keep this issue open while I tackle that. Might be a while, hopefully the code is documented well enough for most folks to get a general gist of what to do
